Library/OcBootManagementLib/VariableManagement.c

```diff
@@ -244,6 +246,69 @@ DeleteVariables (
   }
 }
 
+VOID *
+InternalGetBootstrapBootData (
+  OUT UINTN  *OptionSize
+  )
+{
+  EFI_STATUS               Status;
+  UINTN                    BootOrderSize;
+  UINT16                   *BootOrder;
+  UINT16                   OptionIndex;
+  VOID                     *OptionData;
+
+  BootOrderSize = 0;
+  Status = gRT->GetVariable (
+    EFI_BOOT_ORDER_VARIABLE_NAME,
```

Comment:
Do not we really have functions for boot order reading?

Reply:
No, we do it in two places it seems, and above case pre-reserved the first index to avoid re-alloc (in case Bootstrap is not already in there) and the second does not. I don't know whether they could be nicely unified without sacrificing the no-realloc.